README.md

```diff
@@ -89,31 +112,31 @@ maprdb mapr:> find /apps/user --limit 2
 To learn more about the various commands, run `help`' or  help command , for example `help insert`.
```

Comment:
Change to "help \<command\>" to make it more clear that "command" should be replaced with a command name.